### PR TITLE
[ONNX] Frontend refactoring

### PR DESCRIPTION
**Pull request by gkrivor**
### Details:
 - Removed onnx_common namespace
 - Removed ngraph::test namespace
 - Removed usage of ngraph/log.hpp

### Tickets:
 - 125501

### CODE REVIEW

**Review by rkazants — APPROVED** [review]
Should we run ONNX runtime integration jobs?

**Review by vurusovs — APPROVED** [review]

**Comment by gkrivor** [comment]
> Should we run ONNX runtime integration jobs that are skipped in CI?

Good catch, it should be enabled in case of changes in ONNX FE, I've added it to this PR

File: .github/workflows/linux.yml [hunk 1]
```diff
@@ -380,7 +380,8 @@ jobs:
 
   ONNX_Runtime:
     name: ONNX Runtime Integration
```
Comment (rkazants) [inline .github/workflows/linux.yml:382]: Jus for completeness, `linux_arm64.yml` also requires the same change

**Review by p-durandin — APPROVED** [review]

File: .github/workflows/linux.yml (hunk 1 above)
Comment (gkrivor) [reply .github/workflows/linux.yml:382]: Added in PR https://github.com/openvinotoolkit/openvino/pull/21931

**Pull request merged**